src/Umbraco.Web.UI.Client/src/common/services/umbdataformatter.service.js

```diff
@@ -365,7 +365,7 @@
                     action: action,
                     variants: _.map(displayModel.variants, function(v) {
                         return {
-                            name: v.name,
+                            name: v.name ? v.name : "", //if its null/empty,we must pass up an empty string else we get json converter errors
```

Comment:
@Shazwazza I think this also can be written as `name: v.name || ""`

Reply:
@bjarnef ah of course, i always forget about that operator in JS thanks!